helmfile/values/harbor.yaml.gotmpl

```diff
@@ -43,22 +50,20 @@ persistence:
       authversion: {{ .Values.citycloud.authVersion }}
       tenant: {{ .Values.citycloud.tenantName }}
       domain: {{ .Values.citycloud.projectDomainName }}
-    {{ else }}
-    {{ if eq .Values.objectStorage.type "s3" }}
+    {{ else if eq .Values.harbor.persistence.type "s3" }}
     type: s3
     s3:
       region: {{ .Values.objectStorage.s3.region }}
       regionendpoint: {{ .Values.objectStorage.s3.regionEndpoint }}
       bucket: {{ .Values.objectStorage.buckets.harbor }}
       secretkey: {{ .Values.objectStorage.s3.secretKey }}
       accesskey: {{ .Values.objectStorage.s3.accessKey }}
-    {{ else if eq .Values.objectStorage.type "gcs" }}
+    {{ else if eq .Values.harbor.persistence.type "gcs" }}
```

Comment:
I think we want to have this set to `objectStorage` or something which took the global object storage settings? Because it's very strange to needing to setup the bucket name/region etc. in the global object storage and then set the type here

Reply:
I get your point. Though the start of this `if else` chain has `{{ if eq .Values.harbor.persistence.type "filesystem" }}`, that's why I used the same here.

One alternative would be to have `harbor.persistence.type` use either `filesystem` or `objectstorage` and pick the exact type of object storage from the global config. Would that work?

Reply:
The only thing I thought there was that, what if you wanted to have harbor on s3 and all backups on gcs, then that wouldn't be possible.
But I think that the `filesystem`/`objectstorage` is the best for now, and if there's need for it we can always add the possibility to have different objectstorage backends when we actually have a usecase for it.

Reply:
Ah, I didn't even consider supporting multiple object storages in the cluster. 
As you say I think we will leave that for later. Then I'll make the change suggested.